lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -647,7 +671,7 @@ def _create_course(self, *course_modes, **kwargs):
             modulestore().update_item(course, ModuleStoreEnum.UserID.test)
 
         for course_mode in course_modes:
-            min_price = (self.MIN_PRICE if course_mode != "honor" else 0)
+            min_price = (0 if course_mode in ["honor", "audit"] else self.MIN_PRICE)
```

Comment:
@rlucioni We updated the logic to use min_price to distinguish between the paid course and free course instead of hard coded mode slugs. So these test cases were using min_price=self.MIN_PRICE for "audit" mode that breaks our new logic. I updated the code to use min_price=0 for "audit" mode.